core/src/main/java/nu/yona/server/properties/SecurityProperties.java

```diff
@@ -5,8 +5,9 @@
 	private int newDeviceRequestExpirationDays = 1;
 	private int passwordLength = 32;
 	private long bruteForceBlockMinutes = 60;
-	private long dosMemorySeconds = 300;
-	private boolean isDOSProtectionEnabled = false;
+	private long dosProtectionWindowSeconds = 300;
+	private boolean isDosProtectionEnabled = false;
+	private int maxCreateUserAttemptsPerTimeWindow = 1;
```

Comment:
I did that intentionally. The other scenario would be running into a failure and retrying. Twice in 5 minutes sounds reasonable for that. Agreed?